### ISSUE TITLE
Neo:3.0.0-preview2-07 is connected with itself? (seed2.neo.org)

### ISSUE THREAD

**Issue by wy** [neo-project/neo-cli#215]
Whilst building the latest RPC network graph for Happynodes, we noticed that seed2.neo.org (running Neo:3.0.0-preview2-07) is peered with itself.

```
> ping seed2.neo.org
`Pinging seed2.neo.org [101.132.33.44] with 32 bytes of data:
```

And

```
curl -X POST \
  http://seed2.neo.org:10332/ \
  -d '{
        "jsonrpc": "2.0",
        "method": "getpeers",
        "params": [],
        "id": 3
}'
```

Includes itself as a connected peer:

```
"connected": [
            {
                "address": "101.132.33.44",
                "port": 10333
            },
```

As far as we know, it doesn't make sense to peer with yourself?
Could it be a bug.

![image](https://user-images.githubusercontent.com/1134163/44001328-ba1e9276-9e27-11e8-9c58-b41a1692de54.png)

**Comment by erikzhang** [comment]
This may be because 2 nodes are running on seed2.neo.org, or it may be a bug. Still need to observe.

**Comment by wy** [comment]
Thanks @erikzhang

**Comment by wy** [comment]
![image](https://user-images.githubusercontent.com/1134163/44311833-79702b80-a3e6-11e8-948b-9051f986318b.png)
Seed5 is now doing it, are you running 2 nodes there?

**Comment by jseagrave21** [comment]
![screenshot_20180819-195506](https://user-images.githubusercontent.com/40873301/44318428-aaeaf600-a3ea-11e8-9e64-b1fb4a460166.png)
Another one is doing it now.

**Comment by erikzhang** [comment]
I'm not running 2 nodes on single domain. It seems to be a bug.

**Comment by vncoelho** [comment]
This issue was not detected after released of 2.9.0+, if any problem is still found fell free to reopen.
